docs/source/api-reference/drivers/flashers.md

```
@@ -265,26 +269,26 @@
-| `login.username` | Username to log in with, leave empty if not needed |  |
-| `login.password` | Password for login, leave empty if not needed |  |
-| `login.prompt` | Shell prompt after successful login | # |
-| `preflash_commands` | List of commands to run before flashing, useful to clear boot entries, etc | |
-| `kernel.file` | Path to kernel image within bundle |
-| `kernel.address` | Memory address to load kernel to | |
-| `initram.file` | Path to initramfs within bundle (if any)| |
-| `initram.address` | Memory address to load initramfs to (if any) | |
-| `dtb.default` | Default DTB variant to use | |
-| `dtb.address` | Memory address to load DTB to  | |
-| `dtb.variants` | Map of DTB variant names to files |
+| Field                | Description                                                                | Default |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)